packages/fuels/tests/harness.rs

```diff
@@ -3716,3 +3717,17 @@ async fn test_rust_result_can_be_encoded() -> Result<(), Box<dyn std::error::Err
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_script_interface() -> Result<(), Error> {
+    let path_to_bin = "../fuels/tests/test_projects/logging/out/debug/logging.bin";
+
+    let return_val =
+        run_compiled_script(path_to_bin, TxParameters::new(None, Some(2500), None)).await?;
+
+    let correct_hex =
+        hex::decode("ef86afa9696cf0dc6385e2c407a6e159a1103cefb7e2ae0636fb33d3cb2a9e4a");
+
+    assert_eq!(correct_hex?, return_val[0].data().unwrap());
```

Comment:
```suggestion
        hex::decode("ef86afa9696cf0dc6385e2c407a6e159a1103cefb7e2ae0636fb33d3cb2a9e4a")?;

    assert_eq!(correct_hex, return_val[0].data().unwrap());
```

Reply:
I deleted this test from harness.rs, but within the script.rs tests I adopted your suggestions. Thanx!